assets/src/edit-story/components/panels/design/border/borderWidth.js

```diff
@@ -20,43 +20,47 @@
 import PropTypes from 'prop-types';
 import { useCallback } from 'react';
 import styled from 'styled-components';
-import { rgba } from 'polished';
 import { __ } from '@web-stories-wp/i18n';
 
 /**
  * Internal dependencies
  */
-import { Lock, Unlock } from '../../../../icons';
-import { Row, Numeric, Toggle } from '../../../form';
+import { Row } from '../../../form';
 import { useCommonObjectValue } from '../../shared';
+import {
+  LockToggle,
+  NumericInput,
+  Text,
+  THEME_CONSTANTS,
+} from '../../../../../design-system';
+import { MULTIPLE_DISPLAY_VALUE, MULTIPLE_VALUE } from '../../../../constants';
 import { DEFAULT_BORDER } from './shared';
 
+const INPUT_TOTAL_HEIGHT = 64;
+const INPUT_WIDTH = 44;
+
 const BorderRow = styled(Row)`
   ${({ locked }) => locked && 'justify-content: normal'};
 `;
 
-const BoxedNumeric = styled(Numeric)`
-  padding: 6px 6px;
-  border-radius: 4px;
-  max-width: ${({ locked }) => (locked ? 'initial' : '36px')};
-`;
-
-const Space = styled.div`
-  flex: 0 0 10px;
+const Separator = styled.div`
+  width: 8px;
+  margin: -20px 0 6px;
```

Comment:
@miina, I removed all the spacing and left that to the parent flex element's `space-between`. Keeps all components at the proper size regardless of right scrollbar in panel:

| Without scrollbar | With scrollbar |
|-|-|
| ![Screen Shot 2021-03-06 at 18 32 29](https://user-images.githubusercontent.com/637548/110224701-efa4e280-7eab-11eb-8384-e2f3a9ef60bc.png) | ![Screen Shot 2021-03-06 at 18 32 11](https://user-images.githubusercontent.com/637548/110224702-f16ea600-7eab-11eb-8a05-0faf822630a6.png) |